From f92844f2664f79664b256f7ca40068f061a65af6 Mon Sep 17 00:00:00 2001
From: Jon Mason <jdmason@kudzu.us>
Date: Thu, 26 Jan 2023 11:46:33 -0500
Subject: [PATCH] qemu: arm: make QFW, MMIO implied on qemu-arm

There are instances when one would not want QFW enabled, like running
alternative firmware.  Make this choice optional instead of forced by
using the implied keyword.

Signed-off-by: Jon Mason <jdmason@kudzu.us>
Upstream-Status: Pending [Not submitted to upstream yet]
---
 board/emulation/qemu-arm/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/emulation/qemu-arm/Kconfig b/board/emulation/qemu-arm/Kconfig
index 09c95413a5..4ec0331032 100644
--- a/board/emulation/qemu-arm/Kconfig
+++ b/board/emulation/qemu-arm/Kconfig
@@ -5,8 +5,8 @@ config TEXT_BASE
 
 config BOARD_SPECIFIC_OPTIONS # dummy
 	def_bool y
-	select CMD_QFW
-	select QFW_MMIO
+	imply CMD_QFW
+	imply QFW_MMIO
 	imply VIRTIO_MMIO
 	imply VIRTIO_PCI
 	imply VIRTIO_NET
